exporter.go

```diff
@@ -362,6 +366,10 @@ func (e *Exporter) setEnv(opts ExportOptions, launchMD launch.Metadata) error {
 	return nil
 }
 
+func (e *Exporter) setWorkingDir(opts ExportOptions) error {
+	return opts.WorkingImage.SetWorkingDir(opts.AppDir)
```

Comment:
It might be nice to add a debug statement, similar to here: https://github.com/buildpacks/lifecycle/blob/ee2fd58e6e5c16de1b450f4ddcf819849c6a6d5c/exporter.go#L135